docs/sphinx/using/cudaq/platform.rst

```diff
@@ -104,32 +116,134 @@ should be used.
 
 An example of MPI distribution mode usage in both C++ and Python is given below:
 
-C++
-^^^
+.. tab:: C++
 
-.. literalinclude:: ../../snippets/cpp/using/cudaq/platform/observe_mqpu_mpi.cpp
-    :language: cpp
-    :start-after: [Begin Documentation]
-    :end-before: [End Documentation]
+    .. literalinclude:: ../../snippets/cpp/using/cudaq/platform/observe_mqpu_mpi.cpp
+        :language: cpp
+        :start-after: [Begin Documentation]
+        :end-before: [End Documentation]
 
-.. code-block:: console
+    .. code-block:: console
 
-    nvq++ file.cpp -target nvidia-mqpu
-    mpirun -np <N> ./a.out
+        nvq++ file.cpp -target nvidia-mqpu
+        mpirun -np <N> a.out
 
 
-Python
-^^^^^^
+.. tab:: Python
 
-.. literalinclude:: ../../snippets/python/using/cudaq/platform/observe_mqpu_mpi.py
-    :language: python
-    :start-after: [Begin Documentation]
+    .. literalinclude:: ../../snippets/python/using/cudaq/platform/observe_mqpu_mpi.py
+        :language: python
+        :start-after: [Begin Documentation]
 
-.. code-block:: console
+    .. code-block:: console
 
-    mpirun -np <N> python3 file.py
+        mpirun -np <N> python3 file.py
 
 In the above example, the parallel distribution mode was set to :code:`mpi` using :code:`cudaq::parallel::mpi` in C++ or :code:`cudaq.parallel.mpi` in Python.
 CUDA Quantum provides MPI utility functions to initialize, finalize, or query (rank, size, etc.) the MPI runtime. 
 Last but not least, the compiled executable (C++) or Python script needs to be launched with an appropriate MPI command, 
 e.g., :code:`mpirun`, :code:`mpiexec`, :code:`srun`, etc.
+
+Multi-GPU QPU simulation
++++++++++++++++++++++++++++
+
+As shown in the above examples, the :code:`nvidia-mqpu` platform enables
+multi-QPU distribution whereby each QPU is simulated by a single NVIDIA GPU.
+
+In use cases where the number of qubits requires
+the multi-node multi-GPU simulator (:code:`nvidia-mgpu`) as the simulated QPU, 
+multi-QPU distribution capability is provided by the remote simulator target (:code:`remote-sim`).
+
+Specifically, this :code:`remote-sim` platform encapsulates simulated QPUs as independent
+HTTP REST server instances (:code:`cudaq-qpud`). 
+
```

Comment:
Reading only the documenation, I wouldn't know 
1) That I can use the remote-sim/remote-mqpu platform can be used locally without manually launching each server.
2) How to use this platform from Python. 

I would hence suggest to add a code snippet here with a C++ and a Python tab that illustrates the simples way to use the platform with the autolaunch features and explain what it does, then introduce the motivation for the rest of the section with a sentence along the lines of "To customize how many and which GPUs are used for simulating each virtual QPU, one can launch each server manually. For instance, on a manchine with 8 NVIDIA GPUs, ..."

Reply:
Please see https://github.com/NVIDIA/cuda-quantum/pull/1012#discussion_r1458122544 for context about the removal of auto-launch documentation and the latest revision according to the above suggestion.